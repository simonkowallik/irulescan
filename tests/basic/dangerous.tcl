set one 1
expr 1 + $one
