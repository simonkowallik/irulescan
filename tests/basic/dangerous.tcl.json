[{"filepath":"dangerous.tcl.json","warning":[],"dangerous":[]}]
