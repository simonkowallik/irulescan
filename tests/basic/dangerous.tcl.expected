WARNING: Unquoted expr at `1` in `expr 1 + $one`
WARNING: Unquoted expr at `+` in `expr 1 + $one`
DANGEROUS: Dangerous unquoted expr at `$one` in `expr 1 + $one`

